Lib/test/test_ctypes/test_numbers.py

```diff
@@ -146,7 +146,8 @@ def test_floats(self):
     @unittest.skipUnless(hasattr(ctypes, "c_double_complex"),
                          "requires C11 complex type")
     def test_complex(self):
-        for t in [ctypes.c_double_complex]:
+        for t in [ctypes.c_double_complex, ctypes.c_float_complex,
+                  ctypes.c_longdouble_complex]:
```

Comment:
Please use `with subTest(type=t):` to distinguish what failed if a failure occurs.